tests/new_unit_tests/Commands/Connection/ConnectionCommandTest.php

```diff
@@ -0,0 +1,41 @@
+<?php
+/**
+ * @file
+ *
+ * Contains Pantheon\Terminus\UnitTests\Commands\Connection\ConnectionCommandTest
+ */
+
+namespace Pantheon\Terminus\UnitTests\Commands\Connection;
+
+
+use ReflectionMethod;
+
+abstract class ConnectionCommandTest extends \PHPUnit_Framework_TestCase
```

Comment:
Testing the commands via unit tests is, I agree, a little duplicative. However, using PHPUnit allows us to test the command classes themselves and ensure that their constituent parts are in place and working properly. When we can see a specific test for one or more of the specific actions/functions therein failing, it is a much simpler matter to diagnose and fix the error than it would be by seeing the feature test fail and having to dissect the issue.